packages/menus/src/containers/MenuContainer.js

```diff
@@ -575,10 +575,10 @@ class MenuContainer extends ControlledComponent {
             );
 
             if (appendToNode) {
-              menu = <Portal node={appendToNode}>{menu}</Portal>;
+              return <Portal node={appendToNode}>{menu}</Portal>;
```

Comment:
So trying this code with react@15.x.x and supplying the `appendToNode={document.body}` prop wipes out everything in body and renders only the menu? Am I doing something silly?

### Gif
![appendtonode](https://user-images.githubusercontent.com/143402/41334232-286f4a88-6f28-11e8-8337-db2d4faef62b.gif)

Reply:
This was something I noticed as well, it looks like a quirk of the `react-portal` library.

In React 16, since they are using the provided portals, it will append to the body if you explicitly pass it through.  While in React 15 it will completely replace it. This difference sucks, but actually makes sense. React docs even explicitly say to _not_ apply directly to the body.

If we don't provide the `node` prop it is correctly placed within a div relative to the `body` in both 15 and 16 so I removed the previous default.